keyboards/rotary_numpad/info.json

```
@@ -0,0 +1,67 @@
+{
+    "manufacturer": "rabijl",
+    "keyboard_name": "rotary_numpad",
+    "maintainer": "rabijl",
+    "url": "",
+    "usb": {
+        "device_version": "1.0.0",
+        "pid": "0x0000",
+        "vid": "0xFEED"
+    },
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true,
+        "encoder": true,
+        "encoder_map": true
```

This cannot be enabled at the keyboard level, as it depends on keymap level specific configuration.
```suggestion
        "encoder": true
```

---

noted, i'll make a rules.mk for the keymap. Is encoder mapping allowed in the default keymap, or should I make a secondary keymap?

---

> Is encoder mapping allowed in the default keymap, or should I make a secondary keymap?

Encoder mapping is definitely allowed in default mapping, plenty occurrences of it in repo 😉